proto/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "raft-proto"
+version = "0.1.0"
+authors = ["The TiKV Project Developers"]
+edition = "2018"
+license = "Apache-2.0"
+keywords = ["raft", "distributed-systems", "ha"]
+repository = "https://github.com/pingcap/raft-rs"
+homepage = "https://github.com/pingcap/raft-rs"
+documentation = "https://docs.rs/raft-proto"
+description = "Protocol definitions for the rust language implementation of the Raft algorithm."
+categories = ["algorithms", "database-implementations"]
+build = "build.rs"
+
+[features]
+default = []
+gen = []
+
+[build-dependencies]
+protobuf-build = "0.6"
+
+[dependencies]
+bytes = "0.4.11"
+lazy_static = "1.3.0"
```

Comment:
Why does it depend on lazy_static?

Reply:
Some of the rust-protobuf API we are emulating requires an initialised default with static lifetime, lazy_static is the best way to support that.

Reply:
Interesting. It may also be the reason why it doesn't perform as expected.

Reply:
Could `const fn` help solve it one day?

Reply:
Maybe, I'm not sure. I hope we can just eliminate uses of that wrapper function and then remove lazy static